packages/contracts-bedrock/src/deploy-utils.ts

```diff
@@ -366,7 +366,7 @@ export const liveDeployer = async (opts: {
   hre: HardhatRuntimeEnvironment
   disabled: string | undefined
 }): Promise<boolean> => {
-  if (!!opts.disabled) {
```

Comment:
Perhaps parse the env var into a bool higher up the call stack and have `disabled` be a bool arg

Reply:
@tynes done